File: 0/0/1/001.hs
-- Copyright (C) 2013 Jorge Aparicio

main :: IO()
main
  = print
  . sum
  . takeWhile (<upperLimit)
  . filter (\x -> x `rem` 3 == 0 || x `rem` 5 == 0)
  $ positiveIntegers
  where positiveIntegers = [1..] :: [Int]
        upperLimit = 1000

File: 0/0/1/001.cpp
// Copyright (C) 2013 Jorge Aparicio

#include <iostream>

using namespace std;

main() {
  int acc = 0;

  for (int i = 1; i < 1000; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      acc += i;
    }
  }

  cout << acc << endl;

  return 0;
}
